src/Apps/Artwork/Components/ArtworkSidebar2/ArtworkSidebar2CommercialButtons.tsx

```diff
@@ -287,10 +289,50 @@ const ArtworkSidebar2CommerialButtons: React.FC<ArtworkSidebar2CommercialButtons
       return null
     }
 
+    return (
+      <ArtworkSidebarCreateAlertButtonFragmentContainer artwork={artwork} />
+    )
+  }
+
+  const renderButtons = () => {
```

Comment:
We should really avoid this pattern of having a 'render' function inside of another render function. Ideally should be extracted to its own component. If you really don't want to do that then wrap in `useMemo`. (It's caused subtle bugs in production in the past.)